dev/README_RELEASE_PROVIDERS.md

```diff
@@ -757,7 +757,7 @@ script to verify that all expected files are present in SVN. This script will pr
 may help with verifying installation of the packages.
 
 ```shell script
-# Copy the list of packages (pypi urls) into `packages.txt` then run:
+# Copy the list of packages (pypi urls) from email into a `packages.txt` then run:
```

Comment:
Maybe make it also into an actual step to do ?

When it's in the comment I subconsciously don't read it when I am following instructions.

Reply:
Yeah sure, in a bit :)

Reply:
@potiuk I pushed another change as well to make it somewhat explicit.